broker-node/app/Http/Controllers/UploadSessionController.php

```diff
@@ -97,20 +98,22 @@ public function update(Request $request, $id)
             ->first();
 
         // Error Responses
-        if (empty($data_map)) {
-            return response('Datamap not found', 404);
-        }
-        if ($data_map['hash'] != $chunk['hash']) {
-            return response('Forbidden', 403);
-        }
+        if (empty($data_map)) return response('Datamap not found', 404);
+
+        $trytes = new Trytes(["characters" => Trytes::IOTA]);
+
```

Comment:
When I tested Tuupola/trytes, I tested it without passing a param to the constructor.  It's probably fine this way, but just wanted to note that in case we see weird results later.